docs/workflow/sdk-server-overview.mdx

```diff
@@ -39,7 +39,7 @@ Extending VertiGIS Studio Workflow Server requires you to produce a separate ass
 
 -   This will copy the referenced nuget assemblies to the build output folder, which will be important for later.
 
-1. [Add a reference](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager?view=vs-2019) to the `Geocortex.Workflow.Runtime` assembly.
+1. [Add a reference](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager?view=vs-2019) to the `VertiGIS.Workflow.Runtime` assembly.
    By default, this is located in`C:\Program Files\Latitude Geographics\VertiGIS Studio Workflow\Deployment\service`
```

Comment:
Can we update the company name to "VertiGIS" while we are here?

Reply:
There are a few instances of "Latitude Geographics" that can be changed.

Reply:
Were you referring to the file paths? I think I got them all. I also change an old latitudegeo email address to yours. Let me know if you'd rather it was something else. I don't know if there's an appropriate email address for "the Dev Center project".